vendor/bat-native-ads/include/bat/ads/ad_content.h

```diff
@@ -45,7 +45,7 @@ struct ADS_EXPORT AdContent {
   std::string brand_display_url;
   std::string brand_url;
   LikeAction like_action = LikeAction::kNone;
-  ConfirmationType ad_action = ConfirmationType::kUnknown;
+  ConfirmationType ad_action;
```

Comment:
Removing default init value? Curious as to why?

Reply:
Removed unknown type as no longer required